### PR TITLE
Feature map type

### PR DESCRIPTION
**Pull request by bpeng**

### CODE REVIEW

**Comment by junghao** [comment]
1. Should we change the filename of handler.go to something else? The functions in it are not related to handler.
2. I'll recommend you split this PR into 2 PRs. The first is about mtr-api. After the api part has been deployed then create a pr for mtr-ui. In this way you don't have to put temporary codes in commits.

File: mtr-ui/map_page.go [hunk 1]
```diff
@@ -33,13 +34,35 @@ func mapPageHandler(r *http.Request, h http.Header, b *bytes.Buffer) *weft.Resul
...
+				break
+			}
+		}
+	} else {
+		s1 := strings.TrimPrefix(s, "/")
+		for _, mapdef := range p.Border.MapList {
+			if ArrayContains(s1, mapdef.TypeIDs) {
+				p.TypeID = s1
+				p.MapApiUrl = mapdef.ApiUrl
+				typeExist = true
```
Comment (junghao) [inline mtr-ui/map_page.go]: Add a break here?

Comment (bpeng) [inline mtr-ui/map_page.go]: Yes, that will be better

**Comment by bpeng** [comment]
The file ```handlers_auto.go``` is auto generated and I only changed the ```weft.toml``` file.

Splitting to two PRs makes sense to me.

**Pull request closed without merging**